### PR TITLE
Develop MOYO header 

### PR DESCRIPTION
**Pull request by Voloshynd**
- [DEMO LINK](https://Voloshynd.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Voloshynd.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,61 @@
...
+}
+
+.nav__bar {
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  font-size: 12px;
+  font-weight: 500;
```
Comment (anastasiiavorobiova) [inline src/style.css:27]: Consider removing default margins here

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great start! Keep doing &#128293;

File: src/index.html [hunk 2]
```diff
@@ -7,8 +7,34 @@
...
+        <img
+        src="./images/logo.png"
+        alt="logo-image"
+        >
+      </a>
+      <nav class="nav__bar">
+        <ul class="nav__list">
+          <li class="list__item"><a href="#apple" class="is-active nav__link">Apple</a></li>
+          <li class="list__item"><a href="#samsung" class="nav__link">Samsung</a></li>
+          <li class="list__item"><a href="#smartphones" class="nav__link">Smartphones</a></li>
```
[the remark this answers is not in the document]
Comment (Voloshynd) [reply src/index.html]: done

File: src/index.html [hunk 3]
```diff
@@ -7,8 +7,34 @@
...
+          <li class="list__item">
+            <a href="#" data-qa="hover"
+            class="nav__link">Laptops & Computers</a>
```
[the remark this answers is not in the document]
Comment (Voloshynd) [reply src/index.html]: done

File: src/index.html [hunk 4]
```diff
@@ -7,8 +7,34 @@
...
+          </li>
+          <li class="list__item"><a href="#gadgets" class="nav__link">Gadgets</a></li>
+          <li class="list__item"> <a href="#tablets" class="nav__link">Tablets</a></li>
+          <li class="list__item"> <a href="#photo" class="nav__link">Photo</a></li>
+          <li class="list__item"> <a href="#video" class="nav__link">Video</a></li>
```
[the remark this answers is not in the document]
Comment (Voloshynd) [reply src/index.html]: done

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,61 @@
...
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  --active-link-color: #00acdc;
+  --text-color: #000;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```
[the remark this answers is not in the document]
Comment (Voloshynd) [reply src/style.css:11]: done

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,61 @@
...
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
```
[the remark this answers is not in the document]
Comment (Voloshynd) [reply src/style.css]: done

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,61 @@
...
+}
+
+.nav__bar {
+  margin-right: 50px;
```
[the remark this answers is not in the document]
Comment (Voloshynd) [reply src/style.css]: done

File: src/style.css (hunk 1 above)
Comment (Voloshynd) [reply src/style.css:23]: done

File: src/index.html [hunk 8]
```diff
@@ -7,8 +7,34 @@
...
+      <a href="#home" class="logo">
+        <img
+        src="./images/logo.png"
+        alt="logo-image"
+        >
+      </a>
+      <nav class="nav__bar">
+        <ul class="nav__list">
+          <li class="list__item"><a href="#apple" class="is-active nav__link">Apple</a></li>
+          <li class="list__item"><a href="#samsung" class="nav__link">Samsung</a></li>
```
[the remark this answers is not in the document]
Comment (Voloshynd) [reply src/index.html]: done

File: src/index.html [hunk 9]
```diff
@@ -7,8 +7,34 @@
...
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+        src="./images/logo.png"
+        alt="logo-image"
+        >
+      </a>
+      <nav class="nav__bar">
+        <ul class="nav__list">
+          <li class="list__item"><a href="#apple" class="is-active nav__link">Apple</a></li>
```
[the remark this answers is not in the document]
Comment (Voloshynd) [reply src/index.html]: done

File: src/index.html [hunk 10]
```diff
@@ -7,8 +7,34 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&family=Oswald:wght@300;600;700&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+        src="./images/logo.png"
+        alt="logo-image"
+        >
```
[the remark this answers is not in the document]
Comment (Voloshynd) [reply src/index.html]: done

File: src/index.html [hunk 11]
```diff
@@ -7,8 +7,53 @@
...
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="logo-image"
+          >
+      </a>
+      <nav class="nav__bar">
```
Comment (sTorba24) [inline src/index.html:22]:
```suggestion
      <nav class="nav">
```

File: src/index.html [hunk 12]
```diff
@@ -7,8 +7,53 @@
...
+        <ul class="nav__list">
+          <li class="list__item">
```
Comment (sTorba24) [inline src/index.html:24]: you don't have block `list` to name element `list__item`
you have block `nav` so item will be `nav__item`

File: src/index.html (hunk 11 above)
Comment (sTorba24) [reply src/index.html:22]: same logic here, it's not element `bar` from block `nav`.
it's block `nav`

File: src/style.css [hunk 13]
```diff
@@ -1,3 +1,59 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  --active-link-color: #00acdc;
+  --text-color: #000;
```
Comment (sTorba24) [inline src/style.css:5]: better to move it to :root selector

**Review by sTorba24 — APPROVED** [review]

File: src/index.html (hunk 12 above)
Comment (Voloshynd) [reply src/index.html:24]: Thanks for feedback

File: src/index.html (hunk 11 above)
Comment (Voloshynd) [reply src/index.html:22]: Thanks for feedback

**Pull request closed without merging**